src/patternfly/components/FormControl/form-control.scss

```
@@ -27,10 +27,9 @@
   // hover
   --#{$form-control}--hover--after--BorderWidth: var(--pf-t--global--border--width--control--hover);
   --#{$form-control}--hover--after--BorderColor: var(--pf-t--global--border--color--hover);
-
-  // focus
-  --#{$form-control}--focus--after--BorderWidth: var(--pf-t--global--border--width--control--clicked);
-  --#{$form-control}--focus--after--BorderColor: var(--pf-t--global--border--color--clicked);
+  --#{$form-control}--hover--m-success--after--BorderColor: var(--pf-t--global--border--color--status--success--hover);
+  --#{$form-control}--hover--m-warning--after--BorderColor: var(--pf-t--global--border--color--status--warning--hover);
+  --#{$form-control}--hover--m-error--after--BorderColor: var(--pf-t--global--border--color--status--danger--hover);
```

super duper nit, we typically name this `--component--variation--hover` so it would be 

```suggestion
  --#{$form-control}--m-success--hover--after--BorderColor: var(--pf-t--global--border--color--status--success--hover);
  --#{$form-control}--m-warning--hover--after--BorderColor: var(--pf-t--global--border--color--status--warning--hover);
  --#{$form-control}--m-error--hover--after--BorderColor: var(--pf-t--global--border--color--status--danger--hover);
```